src/server.c

```diff
@@ -3694,6 +3694,9 @@ void afterCommand(client *c) {
     /* Flush pending tracking invalidations. */
     trackingHandlePendingKeyInvalidations();
 
+    clusterSlotStatsAddNetworkBytesInForUserClient(c);
```

Comment:
I almost wonder if this should be in `commandProcessed` instead of afterCommand. Not strictly a blocker, just a thought since there is special code in multi.c to handle that case.